client/src/components/dashboard/deploy/form/DeployForm.js

```diff
@@ -0,0 +1,88 @@
+import { Formik } from 'formik';
+import * as yup from 'yup';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  Row, InputGroup, Button, Form, Col,
+} from 'react-bootstrap';
+
+import { updateWebsiteTitle } from '../../../../actions/website';
+import { deployToGHPages } from '../../../../actions/team';
+
+const DeployForm = ({ teamId }) => {
+  const { title } = useSelector((state) => state.website);
```

Comment:
I thought we were going to simply embed the team's name for the title in scholly. Any reason that didn't work out? If we're sticking with that approach, I feel like we shouldn't expose code that will not be used. Ditto for the corresponding functionality, including actions, etc.

Reply:
If we're using team name as title, it makes sense to leave in the API functionality in case that requirement arises in the future. But IMO we shouldn't expose UI to allow users to edit it.

Reply:
Oh, I thought we are going to let the custom title default to the orgName (see lines 13 and 24) and the user has the choice to modify it?! 
So you mean we just left the deploy UI the same on Researchicy as before and just grab the orgName in scholly to be the title?

Reply:
Yep that's what I was thinking. Unless we think that there is value in setting the title.
@vgoh1 @kdav108 @rpra0006 @jfan0010

Reply:
I don't see much value in customizing the browser tab title, imo having Team/Org Name by default should be enough.

Reply:
Personally agree with Akshay and Richard, but I don't think Chunyang will give in to this. If you check his GitHub page, he's also customizing the tab title himself.
![image](https://user-images.githubusercontent.com/48777884/132449171-210bdb7b-cd0a-4d95-a9f2-869372f94388.png)

Reply:
Makes sense, only thing with that is that Chunyang's page is a personal reflection of his own research career. Since Researchify is centered around Research **teams** do we still see value in the custom title?

Reply:
I feel given that this is implemented already, there's little weight in rejecting all the effort now. Why not just keep the team name as default and if any researcher (like Chunyang) wants to personalise for SEO reason or otherwise, they can still do it. We can let the user decide.

Also agree with Akshay's comment that Chunyang's argument is more reasonable for personal sites and not exactly team pages.

Reply:
Yes we can keep the functionality but at a UI standpoint isn't it a bit confusing for less technical people when they enter a "Website Title"?. I'd expect the website title to be what people see (actual site) when they visit my website and not the browser tab title.

Reply:
+1. We definitely don't want to diminish anyone's effort, but I do believe our UI will be better off without it. IMO, it's an anti-pattern to leave in functionality that doesn't serve justifiable value.

Reply:
Researchify is centered around Research teams <- there's a chance that solo users will be using it right? Not necessarily just teams.
I'm with Kunj, for SEO reasons as well. And I do agree with allowing customization of the browser tab title be confusing, it could be an 'advanced' option that's hidden away unless the user clicks on it? We can discuss it in our winter meeting on how to handle it properly on Sunday.